javascript/javascript1/week4/Vowel_Count.js

```diff
@@ -0,0 +1,26 @@
+function countVowel(givenString){
+    if(givenString === "" || typeof(givenString) !== "string"){
+        return "error";
+    } 
+
+    let vowelCount = 0;
+    const vowels = ["a", "e", "i", "o", "u"];
+
+    for (let vowel of vowels) {
+        for (let letter of givenString) {
```

Comment:
You only need to loop through the "givenString" once and check if each character is a vowel using "vowels.includes(letter)."